common/viewer.cpp

```diff
@@ -1701,67 +1701,67 @@ namespace rs2
 
             if (streams[stream].is_stream_visible())
                 switch (stream_type)
-            {
                 {
```

Comment:
It is not clear whether the indentation changes are intended .. :)

Reply:
indentation was added after a document align command , I reverted the alignment